lib/matplotlib/axes/_base.py

```diff
@@ -2990,6 +2990,10 @@ def _update_title_position(self, renderer):
                 if bb is None:
                     bb = ax.get_window_extent(renderer)
                 top = max(top, bb.ymax)
+                # take yaxis offset text into account
+                bb = ax.yaxis.get_tightbbox(renderer)
+                if ax.yaxis.offsetText.get_text() and bb.ymax > top:
+                    top = bb.ymax
```

Comment:
```suggestion
                if ax._left_title.get_text() != '':   
                    # take yaxis offset text into account
                    bb = ax.yaxis.get_tightbbox(renderer)
                    if ax.yaxis.offsetText.get_text() and bb.ymax > top:
                        top = bb.ymax
```

Reply:
thank you for the proposal - I only now saw it, have already implemented it in a similar way.

Reply:
Your fix was better...